content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -354,8 +354,49 @@ will choose to evict pods of lowest Priority first.
 
 If you are running a [static pod](/docs/concepts/workloads/pods/#static-pods)
 and want to avoid having it evicted under resource pressure, set the
-`priority` field for that Pod directly. Static pods do not support the
-`priorityClassName` field.
+`priorityClassName` field for that Pod directly. 
+
+**Note:**  
+>The kubelet considers all static pods as critical regardless of their `.spec.priorityClassName`.
```

Comment:
Are you sure about this?
My take is that static Pods can be evicted as just other Pods. They will never get admitted unless the kubelet is restarted.